.github/workflows/native_jni_s3_pytorch.yml

```diff
@@ -206,7 +206,7 @@ jobs:
 
   build-pytorch-jni-arm64-macos:
     if: github.repository == 'deepjavalibrary/djl'
-    runs-on: macos-latest-xlarge
+    runs-on: macos-14 #M1
```

Comment:
We should use lower version os macos

Reply:
If we compile the JNI with macos-14, it can only run on macoss-14+
We can use macos-13 to run test, but compile JNI has to use lowest version.